### PR TITLE
The Eggcellent Update Part 2

### PR DESCRIPTION
**Pull request by GreatBigFailure**
This PR lets conveyors put eggs into egg incubators automatically. It also adds the ability to change output direction for incubators via multitool, eggactly the same way as autolathes. 

**Why?** 

eggcellentcontent

This is the best idea ever

:cl:
 * rscadd: Egg Incubators can now be auto-loaded with a conveyor, their output direction can also be changed with a multitool

### CODE REVIEW

**Comment by ghost** [comment]
reeeeee i wanted to add this feature but i guess i forgot to cover this case

**Comment by GreatBigFailure** [comment]
> reeeeee i wanted to add this feature but i guess i forgot to cover this case

Do it for the gibber. It was next on my conveyor list. Would have been easy but that thing is so much more limited than I thought. Would require half recoding it. Can't even put a chicken in the gibber.

File: code/modules/hydroponics/eggincubator.dm [hunk 1]
```diff
@@ -48,10 +49,21 @@
 /obj/machinery/egg_incubator/attackby(var/obj/item/O as obj, var/mob/user as mob)
 	if(..())
 		return 1
-	if(contents.len >= limit)
-		to_chat(user, "\The [src] has no more space!")
-		return 1
-	if (istype(O,input_path))
+	if(ismultitool(O) && !panel_open)
```
Comment (Kurfursten) [inline code/modules/hydroponics/eggincubator.dm]: The fact that this is now copypasted over so many machines is bad for maintainability. Convert it into a machine flag, please.

Comment (GreatBigFailure) [reply code/modules/hydroponics/eggincubator.dm]: Like, on every machine?

Comment (Kurfursten) [reply code/modules/hydroponics/eggincubator.dm]: Yeah, move it up to machinery, you can use SCREWTOGGLE and WRENCHANCHOR as good examples

Comment (GreatBigFailure) [reply code/modules/hydroponics/eggincubator.dm]: No I mean are you asking me to replace all the instances of this copy paste within this PR? Or just make the flag for this machine specifically?

Comment (Kurfursten) [reply code/modules/hydroponics/eggincubator.dm]: Please do delete earlier pasting in other machines that use it and replace with giving them the flag

Comment (GreatBigFailure) [reply code/modules/hydroponics/eggincubator.dm]: I feel as if that's a little bit much for my silly egg PR but fine fine

Comment (Kurfursten) [reply code/modules/hydroponics/eggincubator.dm]: I'm asking you to do nothing more than take the code you've already pasted here, and move it into machinery, then delete some code blocks and add a flag to other files. In terms of labor that's like three presses of delete, one cut/paste, and about thirty characters of writing based on what you pick as your flag name.

Comment (GreatBigFailure) [reply code/modules/hydroponics/eggincubator.dm]: There's not actually any other machines besides this and R&D machines that have need for this, unless I'm misunderstanding what you mean.

File: __DEFINES/setup.dm [hunk 2]
```diff
@@ -1147,6 +1147,7 @@ var/default_colour_matrix = list(1,0,0,0,\
 #define WIREJACK		512 //can we wirejack it? if flagged, machine calls wirejack()
 #define SHUTTLEWRENCH	1024 //if this flag exists, the computer can be wrenched on shuttle floors
 #define SECUREDPANEL 2048 //it won't let you open the deconstruction panel if you don't have the linked account number. Originally used for custom vending machines
+#define MULTIOUTPUT 4096 //Let's you set the output location with a multitool
```
Comment (Kurfursten) [inline __DEFINES/setup.dm:1150]: let us you... mmm 🤤

Comment (GreatBigFailure) [reply __DEFINES/setup.dm:1150]: No, NO

Comment (Eneocho) [reply __DEFINES/setup.dm:1150]: yaya

**Pull request merged**